packages/vue-flicking/demo/plugins/Autoplay.vue

```diff
@@ -37,7 +37,11 @@ export default class AutoPlayDemo extends Vue {
   }
 
   plugins = [new Fade(), new AutoPlay(2000, "NEXT")];
-  code0 = `<flicking
+  code0 = `<!--
+  import { Fade, AutoPlay } from "@egjs/flicking-plugins";
+  plugins = [new Fade(), new AutoPlay(2000, "NEXT")];
```

Comment:
@WoodNeck

It seems'd better add a `const`.

Reply:
I'll fix that!